docker-compose.override.yml

```diff
@@ -6,7 +6,7 @@ services:
   # etc. To enable it, update .env, set:
   # DOMAIN=localhost.tiangolo.com
   proxy:
-    image: traefik:3.0
+    image: traefik:comte
```

Comment:
I would prefer to have the number here, `3.1` I guess?

Reply:
What about just using the major? `3` instead of your suggested `3.1`? Whatever you say...